spec/main_page_links_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Main page links' do
+  before do
+    @main_page, @test = TestingAppServer::AppServerHelper.new.init_instance
+  end
+
+  after do |_example|
+    @test.webdriver.quit
+  end
+
+  it 'All modules on main page present' do
+    expect(@main_page).to be_all_modules_present
+  end
+
+  it 'All side bar modules open correctly' do
```

Comment:
I think it better idea to split those long test into several very small one

It will be easier to interpreter their result if `only  one case failed because settings module missing`, than `some modules missing, settings and maybe project and all others, but we don't know`